### DISCUSSION TITLE
Issues in oral PK simulation

### DISCUSSION THREAD

**Discussion by harishkaushikbugworks** [Open-Systems-Pharmacology/Forum#742] [Questions & Problems]
Hello All,
Iam trying to build Prazosin PBPK model for oral pk simulation

1. I have compiled the compound details from the literature
2. I have collected IV human data from literature and did parameter optimization
3. I could get IV Data fit from predictions
4. Following this i have used formulation data (Fraction dissolved) and oral dose details 
5. Here is the issue my oral PK simulation is not matching with observed data

Can any one help me in this

[PRAZOSIN PBPBK.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/3684763/PRAZOSIN.PBPBK.zip)

**Comment by HenrikCordes** [comment]
Dear @harishkaushikbugworks ,

thank you for sharing you work with the comunity!

I'm not faminilar with the drug in you model, I assume the ADME and PhysChem are correct in that model. 
At the first glance I noticed that the intestinal permeability value your PO simulation is the PK-Sim predicted value. 
Do you have any experimental information on intestinal permeability, such as from Caco2 or MDCKII in vitro assays? 

A slight increase of the intestinal permeability (somewhere between 1 and 2 E-6 cm/min) gives you a very nice fit of the PO data. 
Here, other PO doses (higher or lower) would be benefitial to support the intestinal permeability value.

Hope this helps and kind regards,
Henrik

**Comment by harishkaushikbugworks** [comment]
Hello Henrik,

Thanks for your invaluable comments, it really helped me to improve the fitting by changing the permeability values form literature

Drug iam trying to fit is prazosin, iam attaching the fit is got, but still there is some optimisation needed, can you help me as to what are those parameters i have to optimise further

iam attaching fitting curve and the project file as zipfile

Regards
Harish
![prazosin fitting](https://user-images.githubusercontent.com/49284633/66113595-b5fee680-e5ea-11e9-80af-411eb31a5d7a.jpg)
[PRAZOSIN PBPBK.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/3685253/PRAZOSIN.PBPBK.zip)

**Comment by harishkaushikbugworks** [comment]
Hello Experts,

In continuation to my above efforts, i optimized the fitting but facing difficulty in fitting the oral IR capsule data of test and reference, can anyone please help me by giving suggestions

Iam attaching the project file for the same

Any help is greatly needed

Regards
Harish
[PRAZOSIN PBPBK.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/3690132/PRAZOSIN.PBPBK.zip)
